WordPress/Classes/Models/ManagedPerson.swift

```diff
@@ -11,9 +11,9 @@ class ManagedPerson: NSManagedObject {
         firstName = person.firstName
         lastName = person.lastName
         role = String(person.role)
-        siteID = Int32(person.siteID)
-        userID = Int32(person.ID)
-        linkedUserID = Int32(person.linkedUserID)
+        siteID = Int64(person.siteID)
+        userID = Int64(person.ID)
+        linkedUserID = Int64(person.linkedUserID)
```

Comment:
Huh i did add that change, but it's in another branch. Adding it right here, right now. Nice catch!